index.html

```diff
@@ -1,84 +1,111 @@
 <!DOCTYPE html>
-<meta charset="utf-8">
-<meta name="viewport" content="width=device-width, initial-scale=1.0">
-<title>tetick.xyz</title>
-<link rel="icon" href="favicon.ico" type="image/x-icon">
-<link rel="stylesheet" href="style.css">
-<link rel="stylesheet" href="awesomplete.css">
-<div class="header">
-<a id="index">tetick.xyz</a>
-<a id="save">save</a>
-<a id="about-link">about</a>
-<a id="wrong-link">wrong!</a>
-</div>
+<html lang="en">
```

Comment:
there is no point serving those, the browser injects them. see google's [style guide](https://google.github.io/styleguide/htmlcssguide.xml).